stable/appmesh-grafana/dashboards/control-plane.json

```diff
@@ -1,415 +1,681 @@
 {
-    "annotations": {
-        "list": [
-            {
-                "builtIn": 1,
-                "datasource": "-- Grafana --",
-                "enable": true,
-                "hide": true,
-                "iconColor": "rgba(0, 211, 255, 1)",
-                "name": "Annotations & Alerts",
-                "type": "dashboard"
-            }
-        ]
+  "annotations": {
+    "list": [
+      {
+        "builtIn": 1,
+        "datasource": "-- Grafana --",
+        "enable": true,
+        "hide": true,
+        "iconColor": "rgba(0, 211, 255, 1)",
+        "name": "Annotations & Alerts",
+        "type": "dashboard"
+      }
+    ]
+  },
+  "description": "AWS App Mesh control plane overview",
+  "editable": true,
+  "gnetId": null,
+  "graphTooltip": 0,
+  "links": [],
+  "panels": [
+    {
+      "aliasColors": {},
+      "bars": false,
+      "dashLength": 10,
+      "dashes": false,
+      "datasource": null,
```

Comment:
Isn't the datasource prometheus for this? and for all the metrics here in general?

Reply:
datasource: null means use the datasource marked as default. We don't want to hardcode to a specific source name as the name can be changed or customer may have their own Prometheus configured

Reply:
Yes, but my concern was that some point to prometheus and some point to null

Reply:
Fixed it. There were leftovers